features/google/src/views/ministories/singleDataStory.jsx

```diff
@@ -0,0 +1,12 @@
+import Story from "./story.jsx";
+
+export class SingleDataStory extends Story {
+    constructor(props, neededDataKey) {
+        super(props);
+        this._neededAnalyses = [neededDataKey];
+    }
+
+    get analysisData() {
+        return this.analyses[this._neededAnalyses[0]];
```

Comment:
Then you use the first element... can't this just be `_neededAnalysis`, as in a single one, and

```js
this._neededAnalysis = neededDataKey
```

Also, this is static, it can be computed in the constructor. It could be

```js
this._data = this.analyses[neededDataKey]
```

And simply return it here?

Reply:
If you mean in this PR; I'm good with that. Would appreciate if we took it up later, anyhow.

Reply:
Yeah 👍  just saying bc this is blocking 2 other PRs